### ISSUE TITLE
Modify product properties error

### ISSUE THREAD

**Issue by babyfire** [spree/spree#11893]
## Context

When modifying product properties, a blank property will be added automatically.

## Expected Behavior

Do not automatically add product properties，add it manually instead.

## Steps to Reproduce

![edit](https://github.com/spree/spree/assets/26229604/e67f8c11-cbe7-4621-bd27-2add71893c60)

## Your Environment

* Version used:
  spree (= 4.5.3)
  spree_auth_devise (= 4.5.0)
  spree_backend (= 4.5.1)
  spree_emails (= 4.5.3)
  spree_frontend (= 4.5.0)
  spree_gateway (= 3.10.0)
  spree_i18n (= 5.2.0)
  spree_sample (= 4.5.3)

**Comment by masoud-arabi** [comment]
Hi, Is this issue still open? I am new to open source and I would like to contribute to this issue.

**Comment by rafalcymerys** [comment]
@masoud-arabi yes, it's still open. Actually I think we could just remove the frontend validation for that field - if it's empty it won't add a new property anyway.
